src/mongocxx/pool.hpp

```diff
@@ -53,14 +53,13 @@ class MONGOCXX_API pool {
     ///
     /// @param mongodb_uri
     ///  A MongoDB URI representing the connection parameters
-    /// @param ssl_options
-    ///  Optional SSL options to use when connecting to the MongoDB deployment.
-    ///
-    /// @throws mongocxx::exception if SSL is enabled and ssl_options are
-    /// invalid, or if SSL is not enabled and ssl_options is engaged.
+    /// @param options
+    ///  Client options to use when connecting to the MongoDB deployment.
     ///
+    /// @throws mongocxx::exception if invalid options are provided (whether from the URI or
+    ///  provided client options).
     pool(const uri& mongodb_uri = mongocxx::uri(),
```

Comment:
Since this ctor can legally be called with one argument, I think it should be explicit.